### ISSUE TITLE
FRB should respond to Gum.exe failing to generate fonts

### ISSUE THREAD

**Issue by vchelaru** [vchelaru/FlatRedBall#1578]
If the user does not have XNA installed, then Gum crashes when trying to generate fonts:

```
Application: Gum.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileNotFoundException
   at XnaAndWinforms.GraphicsDeviceControl..ctor()
   at Gum.Wireframe.WireframeControl..ctor()
   at Gum.MainWindow.CreateWireframeControl()
   at Gum.MainWindow..ctor()
   at Gum.Program.Main()
```

Gum should return an error code on exception. FRB should receive that error code and it should properly print output indicating that Gum crashed, and that the user should verify that XNA runtimes exist.

**Comment by nightblade9** [comment]
This is definitely a "read the manual" kind of error ...

Since Gum crashed, I'm not sure if an error code would work here. If you want to proactively check this on start-up of FRB, you could run Gum in a try/catch, and if you catch that specific exception, report to the user that Gum crashed and that they need to install the XNA redistributable.

**Comment by vchelaru** [comment]
This fix has two parts:

1. The next Gum release checks for the exception thrown above. If so, it returns an error code of 2 which means XNA is not installed https://github.com/vchelaru/Gum/commit/267bb4c3cc4d3099772a7db6172e4868b9f2b65f
2. FRB now checks the error code coming from Gum. If it's a 2, then it prints this message: 
![image](https://github.com/user-attachments/assets/2e8514eb-cf01-48e2-9269-524982b2712d) Fixed by 646b3cf1b04311c13575127de28ecc3fc499b123



This is much better than the silent failure from before.
